scene/3d/baked_lightmap.cpp

```diff
@@ -316,7 +316,7 @@ bool BakedLightmap::_bake_time(void *ud, float p_secs, float p_progress) {
 		int mins_left = p_secs / 60;
 		int secs_left = Math::fmod(p_secs, 60.0f);
 		int percent = p_progress * 100;
-		bool abort = bake_step_function(btd->pass + percent, btd->text + " " + itos(percent) + "% (Time Left: " + itos(mins_left) + ":" + itos(secs_left) + "s)");
+		bool abort = bake_step_function(btd->pass + percent, btd->text + " " + itos(percent) + "% (" + RTR("Time Left: ") + itos(mins_left) + ":" + itos(secs_left).lpad(2, "0") + "s)");
```

Comment:
No trailing space in localized strings please :)

Reply:
Fine :P

Reply:
That string is kind of bad in English and needs further localizability though. In French percentages are preceded by a space (`10 %`), and the current time indication is bad in all languages (`12:42s`). Maybe `12'42"` would be better, or just `12:42`.

Reply:
> `12'42"`

Eh, please don't. This notation should be used only for angles, not for time.

Reply:
Fair enough :)